common/json_validation.py

```diff
@@ -0,0 +1,177 @@
+"""
```

Comment:
This duplicates what is in `relation_engine_server/utils/json_validation.py` -- correct? Did you move that file to here and git doesn't realise what's happened, or is this a copy?